micmac_li3ds/api.py

```diff
@@ -188,14 +189,120 @@ def get_or_create_log(self, typ, obj, parent={}):
         self.log.debug(json.dumps(obj, indent=self.indent))
         return obj
 
-    def get_or_create_transfo(self, transfo, type_, source, target):
+    def get_or_create_sensor(self, name, sensor_type, *, description=None,
+                             serial='', specs=None, sensor_id=None):
+        sensor = {
+            'id': sensor_id,
+            'name': name,
+            'type': sensor_type,
+            'description': description,
+            'serial_number': serial,
+            'specifications': {k: v for k, v in specs.items() if v is not None}
```

Comment:
That is exact. I already fixed it in a follow up branch. I can backport the fix here